src/shared/views/Mapper/index.js

```diff
@@ -432,9 +438,11 @@ export default class MapperScreen extends React.Component<Props> {
         // in which case, we use it as the tutorial, or fallback onto the default
         // tutorial content based on the project type
         let tutorialName;
-        if (projectObj.tutorialName !== undefined) {
-            tutorialName = projectObj.tutorialName;
+        if (projectObj.tutorialId !== undefined) {
+            tutorialName = projectObj.tutorialId;
```

Comment:
maybe to be really sure you could rename this to `tutorialId` or `tutorialKey`

Reply:
I've renamed everything to `tutorialId` for consistency.